README.md

```diff
@@ -1,26 +1,97 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+<div align="center">
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+# CephFS server proxy operator
 
-Use links instead.
--->
+A [Juju](https://juju.is) operator for proxying exported Ceph filesystems.
 
-# cephfs-proxy-operator
+[![Charmhub Badge](https://charmhub.io/cephfs-server-proxy/badge.svg)](https://charmhub.io/cephfs-server-proxy)
+[![CI](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/ci.yaml/badge.svg)](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/cephfs-client-operator/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#ubuntu-hpc:matrix.org)
 
-Charmhub package name: operator-template
-More information: https://charmhub.io/cephfs-proxy-operator
+</div>
 
-Describe your charm in one or two sentences.
+## Features
 
-## Other resources
+The CephFS server proxy operator enable CephFS client operators to mount exported Ceph filesystems
+on Ceph clusters not managed by Juju. This proxy operator enables Juju users to manually set 
+a CephFS share endpoint that their CephFS client operators need to mount.
 
-<!-- If your charm is documented somewhere else other than Charmhub, provide a link separately. -->
+## Usage
 
-- [Read more](https://example.com)
+#### With a minimal CephFS kernel server
 
-- [Contributing](CONTRIBUTING.md) <!-- or link to other contribution documentation -->
+First, launch a virtual machine using [LXD](https://ubuntu.com/lxd):
 
-- See the [Juju SDK documentation](https://juju.is/docs/sdk) for more information about developing and improving charms.
+```shell
+$ snap install lxd
+$ lxd init --auto
+$ lxc launch ubuntu:22.04 cephfs-server --vm
+$ lxc shell cephfs-server
+```
+
+Inside the LXD virtual machine, set up a CephFS kernel server that exports a filesystem.
```

Comment:
Let's reference microceph directly:

```suggestion
Inside the LXD virtual machine, set up [microceph](https://github.com/canonical/microceph) to export a CephFS filesystem.
```

Friends of the Ceph team say good things about microceph :sweat_smile: